nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw tech stack remove hyperlink shrink nav bar items

# Description
- Updated Tech Stack Styling; added more line-height to tech stack list items;
- Rearranged the list blocks;
- Updated nav-items to a smaller font-size, 1rem; 
- added line-height 1.5rem to nav-item to give more space between nav-items when they are stacked in mobile view

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
1. Pull down code and run site in browser
2. Check size of Nav items (not including CTA button) are smaller. They should be 1rem
3. In mobile view, check that nav-items have taller spacing in the mobile navbar menu
4. Check that Tech Stack list items have taller spacing between list items
5. Check that Tech Stack items are not links 

ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
